public/js/tracking.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/* global ga */
+
+var settings = require('settings');
+let instance = null;
+
+class Tracking {
```

Comment:
I've made js and jsx both be run through babel see https://github.com/mozilla/payments-ui/pull/157/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR18